apps/dolly-frontend/src/main/js/src/pages/adminPages/Orgtilgang/OrgOversikt.tsx

```diff
@@ -52,6 +53,12 @@ export const OrgOversikt = ({ organisasjonTilgang, loading, error, mutate }: Ove
 													navn={navn}
 													mutate={mutate}
 												/>
+												<RedigerOrganisasjon
```

Comment:
Nå som vi har to knapper i denne kolonnen bør de kanskje ha en hover-tekst for hva de gjør?